framework/src/org/apache/cordova/engine/SystemWebViewClient.java

```diff
@@ -349,7 +348,7 @@ public WebResourceResponse shouldInterceptRequest(WebView view, String url) {
         }
     }
 
-    private static boolean needsKitKatContentUrlFix(Uri uri) {
+    private static boolean needsContentUrlFix(Uri uri) {
```

Comment:
Renaming of this function is a form of cleanup that I think is not super clear from the description or commit messages.

Reply:
Extracted the function rename to a separate commit and updated the PR description.